bin/pycbc_banksim

```diff
@@ -222,6 +223,14 @@ parser.add_argument("--mchirp-window", type=str, metavar="FRACTION",
                          "comma separated numbers to have different bounds "
                          "above and below the signal's, with below bound "
                          "listed first.")
+parser.add_argument("--tau0-window", type=float, metavar="TIME", default=None,
```

Comment:
This option sounds useful but I would have no idea what to choose as a sensible value just looking at the help. Which units does this option assume? Seconds? Is it worth maybe hinting at a reasonable value you found from your own experience in the help?